### DISCUSSION TITLE
Mule account / cubing

### DISCUSSION THREAD

**Discussion by spitfire2112** [isid0re/SoloLeveling#144] [Q&A]
Forgive me if this is already answered somewhere. I didn't see it.

When using a mule account with the main bot, will the main bot go to the mule account to pull items when cubing runewords and stuff?

**Comment by toxickitteh** [comment]
I'm wondering the same thing. Is something like this possible maybe with Limedrop? 

My leveling chars keep running out of space and I'd like to share already made RW with new leveling chars instead of making them again and again or manually dropping them in games etc.

**Comment by tyson-dev** [comment]
As i do it, i use my "Standard Kolbot" to fire up a manual window ( e.g. Profile with d2botmap.dbj as entry script ) and log in with the "sololevel acc". You can then make a game and limedrop items to this char or have your mf'ers join this game to transfer the items over to them which they then will automule accordingly.

I guess you can try setting up automule for sololeveling and post your experiences here?
